common/include/RevCommon.h

```diff
@@ -81,7 +81,7 @@ enum class MemOp : uint8_t {
   MemOpAMO         = 9,
 };
 
-inline uint64_t make_lsq_hash(uint8_t destReg, RevRegClass regType, uint16_t HartID){
+inline uint64_t make_lsq_hash(uint16_t destReg, RevRegClass regType, unsigned HartID){
```

Comment:
`HartID` is `unsigned` in some places and `uint16_t` in others, maybe others. 

I think a `HartID_t` typedef should be defined and used everywhere.

Reply:
HartID should always be unsigned